UserManagement/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM java:8
+FROM maven:alpine
+WORKDIR /app
+COPY . /app
+RUN mvn -v
```

Comment:
do we need to this seeing version thing ?

Reply:
will remove that line